record/record.go

```diff
@@ -42,10 +42,16 @@ func RemapRecord(inputRecord FluentBitRecord, inputTimestamp interface{}, plugin
 		source = "BARE-METAL"
 	}
 	if _, ok = outputRecord["plugin"]; !ok {
-		outputRecord["plugin"] = map[string]string{
-			"type":    "fluent-bit",
-			"version": pluginVersion,
-			"source":  source,
+		if lowDataMode {
+			outputRecord["plugin"] = map[string]string{
```

Comment:
Small data traffic and pipeline optimization: for this case, we don't need to transmit a `map`:
```
"plugin": {
    "type": "k8s-fb-1.9.3"
}
```

We can just transmit a simple `string`:
```
"plugin.type": "k8s-fb-1.9.3"
```

Given that the log ingestion pipeline will do this JSON transformation anyway. By using the "plugin.type" approach, we end up transmitting less data and also avoid having an extra JSON parsing in our pipeline.